apps/storybook/src/stories/datacom-avatar.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { ComponentStoryFn, Meta } from '@storybook/react';
+import { DatacomAvataravatar } from '@datacom/endeavour-react';
+
+export default {
+  title: 'Avatar',
+  component: DatacomAvataravatar,
+  argTypes: {
+    firstName: {
+      name: 'First Name',
+      defaultValue: 'First Name',
```

Comment:
Considering you're setting the values of the args per story, you probably don't need to define these default values.